stanford-earth/matson
# EARTH-1184: Subscribe Link Text and Style Change

# READY FOR REVIEW

# Summary
- Changing up the language in the subscribe link, as well as changing references to Stanford Earth Matters to always include the "Stanford."

# Needed By (Date)
- Next push

# Urgency
- Not very.

# Steps to Test

1. Checkout this branch.
2. Go to a news page that is part of Earth Matters
2. Check that the subscribe stamp starts with "Know your planet" and that the text is not All Caps
3. Check that the small form in the footer to subscribe to Earth Matters starts with "Subscribe to Stanford Earth Matters."

# Affected Projects or Products
- Stanford Earth Matters

# Associated Issues and/or People
- EARTH-1184

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
